sp800-63a/sec10_references.md

```diff
@@ -6,27 +6,25 @@
 
 *This section is informative.*
 
+### 10.1 General References
+
 <a name="A-130"></a>[A-130] OMB Circular A-130, *Managing Federal Information as a Strategic Resource*, July 28, 2016, available at: <https://obamawhitehouse.archives.gov/sites/default/files/omb/assets/OMB/circulars/a130/a130revised.pdf>.
 
-<a name="canada"></a>[Canadian Guideline on Identity Assurance] available at: <http://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=30678&section=HTML>.
+<a name="COPPA"></a>[COPPA] *Children's Online Privacy Protection Act of 1998 ("COPPA")*, 15 U.S.C. 6501-6505, 16 CFR Part 312, available at: <https://www.law.cornell.edu/uscode/text/15/chapter-91>.
 
-<a name="COPPA"></a>[COPPA] Children's Online Privacy Protection Act of 1998 ("COPPA"), 15 U.S.C. 6501-6505, 16 CFR Part 312
+<a name="9397"></a>[EO 9397] Executive Order 9397, *Numbering System for Federal Accounts Relating to Individual Persons*, November 22, 1943, available at: <https://www.ssa.gov/foia/html/EO9397.htm>.
 
 <a name="dmf"></a>[DMF] National Technical Information Service, *Social Security Death Master File*, available at: <https://www.ssdmf.com/Library/InfoManage/Guide.asp?FolderID=1>.
 
 <a name="E-Gov"></a>[E-Gov] *E-Government Act of 2002* \(includes FISMA) (P.L. 107-347), December 2002, available at: <http://www.gpo.gov/fdsys/pkg/PLAW-107publ347/pdf/PLAW-107publ347.pdf>.
 
-<a name="9397"></a>[EO 9397] *Executive Order 9397, Numbering System for Federal Accounts Relating to Individual Persons*, November 22, 1943, available at: <https://www.ssa.gov/foia/html/EO9397.htm>.
-
-<a name="fbcacp"></a>[FBCACP] X.509 Certificate Policy For The Federal Bridge Certification Authority (FBCA), Version 2.27, December 2, 2013, available at: <https://www.idmanagement.gov/IDM/servlet/fileFieldentityId=ka0t0000000TN7cAAG&field=File__Body__s>.
+<a name="fbcacp"></a>[FBCACP] X.509, *Certificate Policy For The Federal Bridge Certification Authority (FBCA)*, Version 2.27, December 2, 2013, available at: <https://www.idmanagement.gov/IDM/servlet/fileFieldentityId=ka0t0000000TN7cAAG&field=File__Body__s>.
```

Comment:
I think X.509 is part of the title and should be inside the asterisks. But in checking this I found that the URL is incorrect.